deployments/common/image/Dockerfile.trailer

```diff
@@ -38,7 +32,20 @@ RUN apt-get update && \
     apt-get dist-upgrade --yes && \
     apt-get clean
 
+# For standalone operation outside JupyterHub,  note that  /etc also includes
+# common home directory files.
+
+RUN tree -a /etc/default-home-contents/
```

Comment:
Dockerfile.trailer and jwebbinar/Dockerfile are slightly out of sync for the "tree" commands.   Any build will sync them up,  but just FYI,   it's good to save Dockerfile* in both common and mission deployments.

Reply:
Fixed.